x/group/module/module.go

```diff
@@ -206,13 +206,30 @@ type GroupOutputs struct {
 }
 
 func ProvideModule(in GroupInputs) GroupOutputs {
+	defaultConfig := group.DefaultConfig()
 	/*
 		Example of setting group params:
-		in.Config.MaxMetadataLen = 1000
-		in.Config.MaxExecutionPeriod = "1209600s"
+		in.Config.MaxExecutionPeriod = "1209600s" 	// example execution period in seconds
+		in.Config.MaxMetadataLen = 1000 			// example metadata length in bytes
+		in.Config.MaxProposalTitleLen = 255 		// example max title length in characters
+		in.Config.MaxProposalSummaryLen = 10200 	// example max summary length in characters
 	*/
 
-	k := keeper.NewKeeper(in.Key, in.Cdc, in.MsgServiceRouter, in.AccountKeeper, group.Config{MaxExecutionPeriod: in.Config.MaxExecutionPeriod.AsDuration(), MaxMetadataLen: in.Config.MaxMetadataLen})
+	if in.Config.MaxMetadataLen != 0 {
```

Comment:
This isn't necessary, let's revert this, it is already checked and updated in the keeper.

Reply:
My first guess was the missing API module replace but I do see it in the go.mod. 🤷🏾‍♂️, works for me locally.